pkg/datapath/linux/ipsec/xfrm_collector_test.go

```diff
@@ -112,44 +113,71 @@ cilium_ipsec_xfrm_error{error="state_sequence",type="inbound"} 7
 cilium_ipsec_xfrm_error{error="state_sequence",type="outbound"} 21
 cilium_ipsec_xfrm_error{error="template_mismatched",type="inbound"} 11
 `
+
+	maxSeqMetric = `
+# HELP cilium_ipsec_xfrm_max_sequence_number Max sequence number of xfrm anti-replay
+# TYPE cilium_ipsec_xfrm_max_sequence_number gauge
+cilium_ipsec_xfrm_max_sequence_number 35308`
 )
 
 func (x *XFRMCollectorTest) Test_xfrmCollector_Collect(c *C) {
 	tests := []struct {
-		name           string
-		statsFn        func() (procfs.XfrmStat, error)
-		expectedMetric string
-		expectedCount  int
+		name                  string
+		statsFn               func() (procfs.XfrmStat, error)
+		maxSequenceNumberFunc func() (uint32, error)
+		expectedMetric        string
+		expectedCount         int
 	}{
 		{
 			name: "error while getting stats",
 			statsFn: func() (procfs.XfrmStat, error) {
 				return procfs.XfrmStat{}, fmt.Errorf("error due to some reason")
 			},
-			expectedCount:  0,
-			expectedMetric: "",
+			maxSequenceNumberFunc: nil,
```

Comment:
Travis build failures look related. Should you be setting a non-`nil` function for `maxSequenceNumberFunc` here?

![image](https://user-images.githubusercontent.com/17001771/212352535-cfd700d7-1608-466c-8272-5ae215c9e060.png)

Looks good otherwise.

Reply:
thank, I modified the code and test passed.